webapp/src/components/Description.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Typography, Link } from "@mui/material";
+import Doc from "components/Icons/Doc";
+const DOCS_URL = "https://servicenow.github.io/azimuth/user-guide";
+
+type Props = {
+  text?: string;
+  link?: string;
+};
+
+const Description: React.FC<Props> = ({ text, link }) => {
+  return (
+    <Typography variant="body2">
+      {text}
```

Comment:
I think it's too error-prone to expect the `text`s defined everywhere to all finish with a whitespace. Maybe we should create that space using a `<Box gap={1}`? It might also give more flexibility for aligning the icon on the text.
```suggestion
    <Box display="flex" gap={1}>
      <Typography variant="body2">{text}</Typography>
```